Sources/TokamakCore/MountedViews/MountedElement.swift

```diff
@@ -117,29 +117,41 @@ public class MountedElement<R: Renderer> {
 }
 
 extension TypeInfo {
-  func injectEnvironment(from environmentValues: EnvironmentValues, into element: inout Any) {
+  func injectEnvironment(from environmentValues: EnvironmentValues, into element: inout Any) -> EnvironmentValues {
```

Comment:
<!--
  1 failure:  Line should be 10...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Sources/TokamakCore/MountedViews/MountedElement.swift;
  Line: 120;
-->
- :no_entry_sign: Line should be 100 characters or less: currently 115 characters (`line_length`)